### PR TITLE
fix test data

### PR DESCRIPTION
**Pull request by mhrimaz**

### CODE REVIEW

File: test_data/rim_test2_output.txt [hunk 1]
```diff
@@ -124,9 +124,21 @@
...
+ ╔════════╤═══╤═══╤═══╤═══╤═══╤═══╤═══╤═══╤═══╤═══════╗ 
 ┌──────────────────────────────────────────────────────┐
-│                     Go team fish!                    │
-│                     Finally free.                    │
+│                    Missing inputs.                   │
+│                    The games ends!                   │
 └──────────────────────────────────────────────────────┘
- ╚════════╧═══╧═══╧═══╧═══╧═══╧═══╧═══╧═══╧═══╧═══════╝
\ No newline at end of file
+ ╚════════╧═══╧═══╧═══╧═══╧═══╧═══╧═══╧═══╧═══╧═══════╝
```
Comment (Kombustor) [inline test_data/rim_test2_output.txt]: There's a trailing whitespace

**Review by Kombustor — CHANGES_REQUESTED** [review]

File: test_data/rim_test2_output.txt (hunk 1 above)
Comment (mhrimaz) [reply test_data/rim_test2_output.txt]: Thanks for checking all the test cases.

**Comment by Kombustor** [comment]
Can be merged, everything is correct @alessiogambi

**Comment by alessiogambi** [comment]
In this case I trust you

**Pull request merged**